_episodes/02-structure.md

```diff
@@ -53,6 +53,9 @@ Let us layout our first module like the following:
 └── .gitignore                      <- Stock helper file telling git what file name patterns to ignore when adding 
 ```
 
+To visualize your project like above you will use "tree"
+(linux) if you do not have tree you can get using $sudo apt-get install tree
```

Comment:
Can we change "$sudo apt-get install tree" to "\`sudo apt-get install tree\`". So that we get highlight like `this`.